### ISSUE TITLE
UI Accessibility - Proper html5 aria support

### ISSUE THREAD

**Issue by Worfle** [solo-io/gloo-portal-issues#199]
- [x] Missing ```<main>``` tag on pages
- [x] ARIA tag usage
- [x] Improper ```<nav>``` element structure
- [x] Missing ```<select>``` name
- [x] Missing ```<button>``` inner text
- [x] Using ```<div>``` usage as a button

**Comment by Worfle** [comment]
Partial list from #196  

Button work may have already been completed in other issue/PR. I know Torey addressed it, will double-check that it is complete

**Comment by Worfle** [comment]
Aria tag usage has been split out into its own Issue to tackle - #200. I thought it would be simple, but we have a lot of menus and submenus in dropdowns hidden throughout the app. I've gotten some in the work for this issue, but it would be silly to hold the rest of this up for what will be an exhaustive search, find and replace for ALL tag usage.
